archive/tar_test.go

```diff
@@ -747,6 +748,13 @@ func TestBreakouts(t *testing.T) {
 	}
 
 	for _, bo := range breakouts {
```

Comment:
I'm sorry if you have answered about that already, but why? Hardlinks themselves are supported in macOS.

Reply:
right, other Hardlink-related tests are fine actually.

I found that now this test won't run with the PR below, so I'll revert this diff for the next.

https://github.com/containerd/containerd/pull/5264#discussion_r601628758